edudoc/guidelines-for-disciplinary-incidents/guidelines-for-disciplinary-incidents.md

```diff
@@ -44,39 +55,47 @@ The WCA Integrity Committee (WIC) is entitled to apply bans and other disciplina
+- If an organizer's behavior falls outside the scope of the [Code of Conduct](wcadoc{documents/Code%20of%20Conduct.pdf}) but is still concerning or disruptive to the community, it is the Delegate’s discretion whether to work with them. For example, if an organizer has made harmful comments on a non-WCA Discord, the Delegate may decide not to work with them.
 - Carefully keep track of competition budgets and actual expenditures that take place, documenting expenditures with receipts when possible. If you believe that an organizer or legal entity is retaining leftover money from competitions for personal gain then please contact the WIC.
 
-## WCA Staff Members {.page-break-before}
+## WCA Volunteers {.page-break-before}
+
+-  If a WCA Volunteer violates or is strongly suspected to violate the Code of Conduct or Code of Ethics, report the instance directly to the WIC.
+- The WIC will evaluate if a further investigation is needed beyond intervention by a superior.
+- Ensure the WIC is also the correct team to contact. The WCA Quality Assurance Committee or the WCA Regulations Committee may be a more appropriate point of contact for some concerns.
+- Reports must not be included in Delegate Reports, even if the information is anonymized.
+- If you need to report a violation of the [Code of Conduct](wcadoc{documents/Code%20of%20Conduct.pdf}) or [Code of Ethics](wcadoc{documents/Code%20of%20Ethics.pdf}) of a WCA Volunteer who is a member of the WIC, report this behavior to the WCA Appeals Committee at [appeals@worldcubeassociation.org](mailto:appeals@worldcubeassociation.org).
+- If you need to report a violation of the [Code of Conduct](wcadoc{documents/Code%20of%20Conduct.pdf}) or [Code of Ethics](wcadoc{documents/Code%20of%20Ethics.pdf}) of a WCA Board Member, it's recommended to contact the [WIC Leader and Senior Members](https://www.worldcubeassociation.org/teams-committees-councils#wic) directly.
 
-- Like any other competitors, report any violations of the Code of Conduct of WCA Staff to the WIC.
-- If you need to report a violation of the Code of Conduct or Code of Ethics of a Staff member who is a member of the WIC, then please report this behavior to the WCA Appeals Committee at [appeals@worldcubeassociation.org](mailto:appeals@worldcubeassociation.org).
+There may be times when a concern or complaint doesn’t clearly fall under any specific team. It might be outside the WIC’s scope, require general mediation, or feel too minor to submit as a formal report (but still important enough not to ignore). In these cases, contacting the Volunteer’s direct superior can be a useful middle-ground approach. For general reference of these kinds of discussions:
```

Comment:
```suggestion
There may be times when a concern or complaint doesn’t clearly fall under any specific team. It might be outside the WIC’s scope, require general mediation, or feel too minor to submit as a formal report (but still important enough not to ignore). In these cases, contacting the Volunteer’s direct superior can be a useful middle-ground approach. For general reference of these hierarchies:
```